scripts/create-s3-buckets.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
there is nothing specific to `bash` in this script (even `set -e` seems portable) so this should be the best default. I usually run `shellcheck` on all scripts to check for this intricacies.